extensions/BMO/template/en/default/pages/etiquette.html.tmpl

```diff
@@ -18,129 +18,105 @@
   #
   # Contributor(s): Stefan Seifert <nine@detonation.org>
   #                 Gervase Markham <gerv@gerv.net>
+  #                 Emma Humphries <ech@emmah.net>
   #%]
 
 [% PROCESS global/header.html.tmpl
    title = "Bugzilla Etiquette"
    style = "li { margin: 5px } .heading { font-weight: bold }" %]
 
 <p>
-  There's a number of <i lang="fr">faux pas</i> you can commit when using
-  [%+ terms.Bugzilla %]. At the very
-  least, these will make Mozilla contributors upset at you; if committed enough
-  times they will cause those contributors to demand the disabling of your
-  [%+ terms.Bugzilla %] account. So, ignore this advice at your peril.
```

Comment:
I do feel that removing sections like this one and the final one, or rewriting them "straight", are removing character from the document, which is sad.